addons/event/tests/test_event_internals.py

```diff
@@ -108,12 +105,11 @@ def test_event_configuration_from_type(self):
 
         # change template to a one with mails -> fill event as it is void
         event_type.write({
-            'use_mail_schedule': True,
             'event_type_mail_ids': [(5, 0), (0, 0, {
                 'interval_nbr': 1, 'interval_unit': 'days', 'interval_type': 'before_event',
                 'template_id': self.env['ir.model.data'].xmlid_to_res_id('event.event_reminder')})],
-            'use_ticket': True,
             'event_type_ticket_ids': [(5, 0), (0, 0, {'name': 'TestRegistration'})],
+            'note': '<p>Event Type Note</p>'
```

Comment:
Not necessary to test it twice

Reply:
Here we are testing if the event type's note is correctly propagated in case it contains non empty HTML, I think that this might be necessary since in the other test we are only testing if changes are not propagated if the event type's note is set to something empty.

Can you tell me what do you think about it?